src/core/Track.cpp

```diff
@@ -720,16 +720,25 @@ void TrackContentObjectView::mousePressEvent( QMouseEvent * me )
 				m_action = ToggleSelected;
 			}
 		}
-		else if( !me->modifiers() )
+		else
```

Comment:
@BaraMGB Should this be `else`, or `else if( !me->modifiers() )`?

Reply:
The difference would be you can't move a tco with pressed shift and/or alt key. Perhaps ``` else if(!me->modifier) ``` is more discriptive.

Reply:
Okay. The latter is consistent with piano roll. Anyway, you changed it to [the former](https://github.com/LMMS/lmms/pull/3649/commits/46e261f738b09d5b36f83ecae44dab0af18d9cd4#diff-cb2a453c979a4777b9330e2b96644f0eR671) in #3649. Was that intentional?

Reply:
No, I tried to make the code simpler. But now I think ```else if(! me->modifiers() ``` is more discriptive. If someone wants to add a new modifier key, he/she can see where to add it in the code. Also I think, it's better to have a functionality only in a certain circumstance (no modifier key is pressed)

Reply:
Okay. Done via 9d4c377.